sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -79,13 +87,24 @@ export interface OperationTracingOptions {
     spanOptions?: SpanOptions;
 }
 
+// @public
+export interface OperationTracingOptionsLike {
+    spanOptions?: OTSpanOptions;
+}
+
 export { OTSpanContext }
 
 export { OTSpanOptions }
 
 // @public
 export function setTracer(tracer: Tracer): void;
 
+// @public
+export interface SpanConfig {
```

Comment:
can we have a more specific name? e.g. `CreateSpanFactoryConfig` or something?

Reply:
Done. Just went with our boring convention of naming it after the function it's for and used 'Args'.

Reply:
Looks like you need to generate this file still though ;)

Reply:
Really done this time. :)